### ISSUE TITLE
[FR] CMake: add prefix "SRT" to all srt cmake options 

### ISSUE THREAD

**Issue by IvanPysmenni** [Haivision/srt#2914]
There is an inconvenience with configuration srt lib build in a case when srt was added to CMake project as subdirectory. SRT options doesn't have an unique name. So during configuration in root CMake file there is a chance that this name is already used by another project in your build(for example ENABLE_APPS). It would be to add prefix "SRT" to all srt cmake options. For example: SRT_ENABLE_APPS. 

As an example this is how it is done in spdlog:
https://github.com/gabime/spdlog/blob/v1.x/CMakeLists.txt#L59

**Comment by ethouris** [comment]
Good idea, just there will have to be still maintained the old option names somehow.

OTOH the configure script may remain with all old names, at least for explicitly defined options. So it would translate `--enable-apps` into `-DSRT_ENABLE_APPS=1` .

**Comment by maxsharabayko** [comment]
I think we can just change them in 1.6.0.
